walleve/walleve/peernet/peernet.cpp

```diff
@@ -32,12 +32,18 @@ void CPeerNet::ConfigNetwork(CPeerNetConfig& config)
     confNetwork = config;
 }
 
+void CPeerNet::AddPeerRecord(CPeer* pPeer)
+{
+    setIP.insert(pPeer->GetLocal().address());
```

Comment:
should be GetRemote() ?

Reply:
@xdwangchn  Fixed this typo of method name